.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,40 @@
+name: Feature Request
+description: Suggestions for new or different behavior.
+labels:
+  - question
```

Comment:
should we add a new label for feature requests? `question` feels a little vague here. also `needs triage` feels like a good label to add too but your call here

```suggestion
  - feature request
  - needs triage
  - question
```

Reply:
what about using the `enhancement` label instead of creating `feature request`?

Reply:
sure!